runtime/autoload/man.vim

```diff
@@ -106,7 +106,7 @@ function! s:system(cmd, ...) abort
     throw printf('command interrupted: %s', join(a:cmd))
   endif
   if opts.exit_code != 0
-    throw printf("command error (%d) %s: %s", jobid, join(a:cmd), opts.stderr)
+    throw printf("command error (%d) %s: %s", jobid, join(a:cmd), opts.stderr[:len(opts.stderr)-2])
```

Comment:
Would be better to remove  whitespace:

    substitute(opts.stderr, '\s\+$', 'g')